p2p_wallet/Scenes/Main/Crypto/Components/Crypto Accounts/Aggregator/CryptoAccountsAggregator.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import KeyAppKitCore
+import Web3
+import Wormhole
+import BigDecimal
+
+struct CryptoAccountsAggregator: DataAggregator {
+    typealias Input = (
+        solanaAccounts: [RenderableSolanaAccount],
+        ethereumAccounts: [RenderableEthereumAccount]
+    )
+    typealias Output = (
+        transfers: [any RenderableAccount],
+        primary: [any RenderableAccount],
+        secondary: [any RenderableAccount]
+    )
+    
+    func transform(input: Input) -> Output {
+        let (solanaAccounts, allEthereumAccounts) = input
+        
+        /// Claimable transfer accounts
+        let transferAccounts = allEthereumAccounts.filter { ethAccount in
+            switch ethAccount.status {
+            case .readyToClaim, .isClaiming:
+                return true
+            default:
+                return false
+            }
+        }
+        
+        /// Ethereum accounts without claimable transfers
+        let filteredEthereumAccounts = allEthereumAccounts.filter { account in
+            return !transferAccounts.contains(account)
+        }
+
+        let mergedNonTransferAccounts: [any RenderableAccount] = (filteredEthereumAccounts + solanaAccounts)
+            .filter(hiddenFilter)
+            .sorted(by: commonSort)
+        
+        let primaryAccounts = mergedNonTransferAccounts
+            .filter(primaryFilter)
+        
+        let secondaryAccounts = mergedNonTransferAccounts
+            .filter { !primaryFilter(account: $0) }
+
+        return (transferAccounts, primaryAccounts, secondaryAccounts)
+    }
+    
+    // MARK: - Helpers
+    
+    // Filter out hidden accounts
```

Comment:
The comments for functions need to be started with three slash symbol (///), not two (//)